doozer/doozerlib/cli/scan_sources.py

```
@@ -81,6 +82,40 @@ async def run(self):
         # We have our information. Now build and print the output report
         await self.generate_report()
 
+    def _is_meta_enabled(self, meta: Metadata) -> bool:
+        return meta.enabled or meta.mode == "disabled" and self.runtime.load_disabled
```

Ruff (0.15.10)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)